src/tile/SpotlightTile.tsx

```diff
@@ -210,7 +212,9 @@ export const SpotlightTile = forwardRef<HTMLDivElement, Props>(
     const ref = useMergedRefs(ourRef, theirRef);
     const maximised = useObservableEagerState(vm.maximised);
     const media = useObservableEagerState(vm.media);
-    const [visibleId, setVisibleId] = useState(media[0].id);
+    const [visibleId, setVisibleId] = useState<string | undefined>(
+      media.length > 0 ? media[0].id : undefined,
```

Comment:
Any reason why this would not work?
```suggestion
      media[0]?.id,
```

Reply:
No. I always forget that JS has this behaviour rather than throwing an array index out of bounds exception.